apps/rush-lib/src/logic/versionMismatch/VersionMismatchFinder.ts

```diff
@@ -76,7 +79,8 @@ export class VersionMismatchFinder {
   ): void {
     VersionMismatchFinder._checkForInconsistentVersions(rushConfiguration, {
       ...options,
-      isRushCheckCommand: false
+      isRushCheckCommand: false,
+      truncateLongPackageNameLists: true
```

Comment:
Means we will be truncating on `rush install`/`rush update` by default. I also don't think we have a `--verbose` parameter there. Is the logic that if you see this error, you should be running `rush check` until it comes back clean and then running `rush install`/`rush update`? I'm okay with that, just wanted to make sure we want to default to this behavior.

Reply:
I suspect that most users aren't familiar with `rush check` and would not realize this.  

Maybe the tool could print a notice such as:

```
For more detailed reporting about these version mismatches, use the "rush check --verbose" command.
```
The notice should be worded in such a way that it makes sense regardless of whether `(and 623 others)`  truncation was needed.